docs/builds/guides/migration/migration-to-27.0.0.md

```diff
@@ -24,3 +24,40 @@ From v27.0.0 the {@link module:clipboard/clipboardpipeline~ClipboardPipeline `Cl
 The {@link module:engine/view/document~Document#event:clipboardInput `view.Document#clipboardInput`} and {@link module:clipboard/clipboardpipeline~ClipboardPipeline#event:inputTransformation `ClipboardPipeline#inputTransformation`} events should not be fired or stopped in your feature code. The `data.content` property should be assigned to override the default content instead. You can stop this event only if you want to completely disable pasting/dropping of some specific content.
 
 You can read about the whole input pipeline in details in {@link framework/guides/deep-dive/clipboard#input-pipeline clipboard pipeline guide}.
+
+## The `view.Document` event bubbling
+
+In v27.0.0 we introduced bubbling of the {@link module:engine/view/document~Document `view.Document`} events, similar to how bubbling works in the DOM. That allowed us to reprioritize many listeners that previously had to rely on the `priority` property. However, it means that existing listeners that use priorities may now be executed at a wrong time (in the different order). The listeners to such events should be reviewed in terms of when they should be executed (in what context/element/phase).
+
+Read more about bubbling events in the {@link framework/guides/deep-dive/event-system#bubbling-events event system guide}.
+
+### The `delete` event
```

Comment:
```suggestion
### The `delete` event
```

Reply:
I wanted to add a missing blank line but GH writer removed it :D

Reply:
:+1: